chart/templates/import-api/configmap.yaml

```diff
@@ -36,6 +36,10 @@ data:
   REDIS_CACHE_PREFIX: "onprem"
   REDIS_HOST: {{ include "carto.redis.host" . }}
   REDIS_PORT: {{ include "carto.redis.port" . }}
+  REDIS_TLS_ENABLED: {{ .Values.externalRedis.tls_enabled | quote }}
+  {{- if and .Values.externalRedis.tls_enabled .Values.externalRedis.tls_ca }}
```

Comment:
This works?
(I said because in my other PR a similar comparison didn't work)

Reply:
It worked when I tested 🤔  Testing it again to be secure

Reply:
Yep, it's working

```yaml
externalRedis:
  host: "whatever.redis.cache.windows.net"
  port: 6380
  password: "whatever"
  tls_enabled: true
  tls_ca: |
    whatever
```

```bash
root@carto-lds-api-67b9fb885f-jvgpc:/usr/src/app# env | grep REDIS
REDIS_PASSWORD=qQzfTMn13LLnFGjbrngSRpkz5pLpbmkMZAzCaKeF0FQ=
REDIS_TLS_CA=whatever
REDIS_HOST=redis-infra-team-9568.redis.cache.windows.net
REDIS_TLS_ENABLED=true
REDIS_PORT=6380
REDIS_CACHE_PREFIX=onprem

```